lunatrace/bsl/frontend/package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "dependencies": {
     "@aws-sdk/client-s3": "^3.52.0",
-    "@lunatrace/lunatrace-common": "link:../common",
+    "@lunatrace/lunatrace-common": "workspace:~",
```

Comment:
the linter seems to be complaining about this. Are you sure this is the correct syntax?